app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<h2 class = "login-heading">Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class = "log-in-form">
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+  </div>
+  
```

Comment:
- Hey, from the wireframe, the inputs and button take up the full width, please, kindly improve on this.

*wireframe*
![image](https://github.com/arnoldnekemiah/blog/assets/59397159/4c4b5516-d1b5-423f-969e-6ef5e9ea1e00)


*your design*
![image](https://github.com/arnoldnekemiah/blog/assets/59397159/96938297-6eca-40d0-af1c-cd9a2d602574)

Reply:
- Kindly make the same changes for your sign_up, the design pattern follows the same wireframe.